src/EnergyPlus/DaylightingManager.cc

```diff
@@ -856,9 +854,14 @@ void CalcDayltgCoeffsRefPoints(EnergyPlusData &state, int const daylightCtrlNum)
     thisDaylightControl.GlareIndexAtRefPt = 0.0; // Glare index at reference points
     thisDaylightControl.SolidAngAtRefPt = 0.0;
     thisDaylightControl.SolidAngAtRefPtWtd = 0.0;
-    thisDaylightControl.IllumFromWinAtRefPt = 0.0;
-    thisDaylightControl.BackLumFromWinAtRefPt = 0.0;
-    thisDaylightControl.SourceLumFromWinAtRefPt = 0.0;
+
+    for (int iExtWin = 1; iExtWin <= thisDaylightControl.TotalExtWindows; ++iExtWin) {
```

Comment:
TIL you cannot do this kind of assignment on Fortran/FCL arrays. Works on `EPVector` though!

Reply:
TIL what "TIL" means!